pageserver/Cargo.toml

```diff
@@ -35,6 +35,8 @@ humantime-serde.workspace = true
 hyper.workspace = true
 itertools.workspace = true
 nix.workspace = true
+# hack to get the number of worker threads tokio uses
+num_cpus = { version = "1.15" }
```

Comment:
I think this functionality is already built-in in std? https://doc.rust-lang.org/stable/std/thread/fn.available_parallelism.html

Reply:
Not the one tokio uses.